webapp/components/notifications/Notification/Notification.vue

```diff
@@ -42,11 +51,26 @@ export default {
   },
   computed: {
     excerpt() {
-      return this.$filters.removeLinks(this.post.contentExcerpt)
+      const excerpt = this.post.id ? this.post.contentExcerpt : this.comment.contentExcerpt
+      return (
+        (!this.post.id ? '<b>Comment: </b>' : '') + excerpt.replace(/<(?:.|\n)*?>/gm, '').trim()
```

Comment:
Can we implement this behaviour in the template please?
```vue
<div v-if="resourceType == 'Post'">
  {{ contentExcerpt | removeHtml }}
</div>
<b v-else>
  {{ contentExcerpt | removeHtml }}
</b>
```

Reply:
I don’t really understand this:

- We add `<b>Comment: </b>` there.
- Therefore the naming `excerpt` should be changed to `contentInformation`. Which I have already done now!
- In your example a `|` operator is used and I can not find its functionality.
- The if and else case has the same code … ??? But you mean `comment.contentExcerpt` for the second one I suppose …

Reply:
Let me explain:
* the `v-else` branch is using `<b>` instead of `<div>`
* the `removeHtml` is a filter that needs to be implemented -it's easy :wink:

Reply:
So I found out and succeed. 👍🏼